include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -135,6 +151,23 @@ class CommonOperatorLoader : public ProtobufLoader {
     nodeValueByName_[op.output(0)] = NodeValue(N, 0);
   }
 
+  void loadMinMax(llvm::StringRef typeName, const OpType &op,
+                      ArgumentDictionaryTy &dict) {
+    const std::string &opName = loadOperatorName(op);
+    auto in0 = getNodeValueOrCreateVariableByName(op.input(0));
+    auto in1 = getNodeValueOrCreateVariableByName(op.input(1));
+
+    Node *node = nullptr;
+    if (typeName == "Min") {
+      node = G_.createMin(opName, in0, in1);
+    }
+    else if (typeName == "Max") {
```

Comment:
Make this just `else`, as it must be `"Max"` if the function is called correctly. Otherwise this gives the impression `node = nullptr` is a possibility.

Reply:
I added a `else` statement with an `assert`. I think this is the safest approach.